src/spotify.rs

```diff
@@ -71,19 +70,18 @@ impl Spotify {
             elapsed: Arc::new(RwLock::new(None)),
             since: Arc::new(RwLock::new(None)),
             channel: Arc::new(RwLock::new(None)),
-            user: None,
         };
 
         let (user_tx, user_rx) = oneshot::channel();
         spotify.start_worker(Some(user_tx));
-        spotify.user = ASYNC_RUNTIME.get().unwrap().block_on(user_rx).ok();
+        let user = ASYNC_RUNTIME.get().unwrap().block_on(user_rx).ok();
         let volume = cfg.state().volume;
         spotify.set_volume(volume);
 
         spotify.api.set_worker_channel(spotify.channel.clone());
         spotify.api.update_token();
 
-        spotify.api.set_user(spotify.user.clone());
+        spotify.api.set_user(user);
```

Comment:
Probably not a problem, but this will set the user later than before. A quick check didn't give me anything, so I think this is fine.

Reply:
I don't fully understand. I thought nothing would change because `Spotify::user` was only used in the `Spotify::new()` method?

Reply:
Huh, yeah, I think I may have misread that. Sorry for the noise!

Reply:
No I understand. I grepped the project so I knew it wasn't used anywhere else. I can imagine you don't know every line off the top of your head :smile:

(PS In the next days I'll probably try to solve/continue work on some of the bugs that have shown up in the last couple of months, if that's ok :slightly_smiling_face:)